pkg/lib/machine-add.html

```diff
@@ -1,17 +1,16 @@
 <div class="modal-header">
     <button class="close" data-dismiss="modal"
         aria-hidden="true">&times;</button>
-    <h4 class="modal-title" translate="yes">Add a New Host</h4>
+    <h4 class="modal-title" translate="yes">Add new host</h4>
 </div>
 
 <div class="modal-body">
     <form class="ct-form">
-      <label translate="yes" class="control-label">Address</label>
+      <label translate="yes" class="control-label">IP address or hostname</label>
```

Comment:
Won't this make the UI too sparse? (that the label will take half of the dialog)
(sure I should just build it and see...)

Reply:
This could also become a tooltip? Honestly, "address" is a very common concept in networking, and pretty much everywhere you can specify a host name or IPv4/IPv6 address.